### PR TITLE
Druid tag is set to 0.20.0 which is latest instead of non-existing tag

### LINKED ISSUE #10792: Druid tag fix at docker-compose.yml
`docker-compose.yml` points Druid version of `0.22.0`. However, latest version in Dockerhub is [0.20.0](https://hub.docker.com/r/apache/druid/tags?page=1&ordering=last_updated).
### Affected Version

0.20.0

### Description

`docker-compose.yml` points Druid version of `0.22.0`. However, latest version in Dockerhub is [0.20.0](https://hub.docker.com/r/apache/druid/tags?page=1&ordering=last_updated). So, you get an image not found error when you try to run project from docker-compose file.

### PR DESCRIPTION
**Pull request by kamaci**
Fixes #10792.

### Description

`docker-compose.yml` points Druid version of `0.22.0`. However, latest version in Dockerhub is [0.20.0](https://hub.docker.com/r/apache/druid/tags?page=1&ordering=last_updated). So, you get an image not found error when you try to run project from docker-compose file.

I've changed the version into latest one, `0.20.0` at docker-compose.yml.

<hr>

This PR has:
- [X] been self-reviewed.
- [X] been tested in a test Druid cluster.

<hr>

##### Key changed/added classes in this PR
 * `docker-compose.yml`

### CODE REVIEW

File: distribution/docker/docker-compose.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ services:
       - ZOO_MY_ID=1
 
   coordinator:
-    image: apache/druid:0.22.0
+    image: apache/druid:0.20.0
```
Comment (suneet-s) [inline distribution/docker/docker-compose.yml:49]: Should we update this to `0.20.1` - now that the latest release was updated

**Comment by suneet-s** [comment]
Thanks for fixing this @kamaci - Did you update the versions in the docker-compose file using some command or by hand? If there's a way to automate this - should we add something to our release process so that we always have the latest?

**Comment by suneet-s** [comment]
Also, sorry for getting around to this so late - I'll be faster on your follow ups to get this in :)

**Comment by jihoonson** [comment]
@kamaci I understand this is confusing since 0.22.0 is a future version as you pointed out, but the version is intentionally set to 0.22.0 to make all versions in the master branch a sort of in sync (https://github.com/apache/druid/blob/master/distribution/asf-release-process-guide.md#preparing-the-master-branch-for-the-next-version-after-branching). We don't publish snapshot images, so docker compose cannot point snapshots. I thought this is OK because there would not be many people who want to use docker based on master. This means, I don't fully understand why we need this change. Can you elaborate more on your motivation?

**Pull request closed without merging**